crates/cargo-remove/src/cargo/ops/cargo_remove/mod.rs

```diff
@@ -45,10 +45,13 @@ pub fn remove(options: &RemoveOptions<'_>) -> CargoResult<()> {
         .dependencies
         .iter()
         .map(|dep| {
-            let section = if dep_table.len() >= 3 {
-                format!("{} for target `{}`", &dep_table[2], &dep_table[1])
-            } else {
-                dep_table[0].clone()
+            let section = {
```

Comment:
nit: *sigh* Feel free to also remove the over-use of iterator combinators.

Reply:
Going to close this fix in a subsequent PR